atuin-client/src/settings.rs

```diff
@@ -20,6 +20,9 @@ pub enum SearchMode {
 
     #[serde(rename = "fuzzy")]
     Fuzzy,
+
+    #[serde(rename = "reorderedfuzzy")]
+    ReorderedFuzzy,
```

Comment:
Honestly I'm happy for this to replace the default fuzzy behaviour! It's nicer :)

Reply:
Just didn't want to step on any toes :) i'll update the PR.